### DISCUSSION TITLE
usage of base @ layer base

### DISCUSSION THREAD

**Discussion by 18121A05L2** [tailwindlabs/tailwindcss#9363] [Help] [answered]
![Screenshot (4402)](https://user-images.githubusercontent.com/59156926/190916257-bcd0ab0d-8271-4554-9678-55e87475f6fa.png)

--> is this valid in tailwind  ?

--> and also i want to add for each property like height , width , padding like things , does this font-size alone in the pic is enough or do we need to add each element and how to add it ?

**Comment by wongjn** [comment, marked as answer]
> is this valid in tailwind ?

It is technically valid, yes.

>  also i want to add for each property like height , width , padding like things , does this font-size alone in the pic is enough or do we need to add each element and how to add it ?

If you are going to be using height, width, padding, etc. utilities with `rem` values, the pixel value of a `rem` unit is dependent on the `html` font size, so this would be enough for these lengths to scale. Otherwise, if this is insufficient for your use case, you could look at adding CSS variables for the properties instead, and change the value of these CSS variables per breakpoint, like you are already doing for `font-size`.

**Reply by 18121A05L2** [reply]
hey bro I applied with this     `<h1 className="text-[3rem] w-[16rem]">dsafsadfas</h1>` but don know why the sizes are constant when when I decreasing from 420 to 320 pixels ?

**Reply by wongjn** [reply]
I presume you have a `@screen mbl` selector at the bottom of all these rules. This will always override the others higher up the CSS file, since in CSS, if two rules have the same specificity, the one that appears later in the file wins. So, I believe you will want to have your breakpoints in reverse order of how you have them now in your CSS file and the `font-size` (and by extension, `rem`) should scale.

**Reply by 18121A05L2** [reply]
yeah , by chainging all to max it is working
